### DISCUSSION TITLE
just loading bar after clicking browse

### DISCUSSION THREAD

**Discussion by halindraprakoso** [minio/minio#16337] [Q&A]
Hello,

long story short, I accidentally deleted my minio container (and do not know which version it was). Fortunately the volume did not get deleted. So I spin up a new minio container using the latest image and mounted the previous volume. The bucket with the usage & number of objects are still there, but after clicking browse, it's just showing the loading bar forever. What can I do?

**Comment by sanjaygeeky** [comment]
Hello I am also getting this error

**Reply by harshavardhana** [reply]
Fix your load balancer to allow websockets
